CHANGELOG.md

```diff
@@ -1,5 +1,133 @@
+* Add retry to ErcotAPI.make_api_call(). [#417](https://github.com/gridstatus/gridstatus/pull/417)
+* Add CAISO Fuel Regions [#426](https://github.com/gridstatus/gridstatus/pull/426)
+* Add ERCOT SPP Day Ahead Hourly to ERCOT API [#425](https://github.com/gridstatus/gridstatus/pull/425)
+* Address Issue with ERCOT LMP Timestamp [#423](https://github.com/gridstatus/gridstatus/pull/423)
+* Adds PJM Ancillary Service Scrapers [#420](https://github.com/gridstatus/gridstatus/pull/420)
+* Add ERCOT SPP Real Time 15 Min to ERCOT API [#408](https://github.com/gridstatus/gridstatus/pull/408)
+* Add spp hourly load dataset scraper [#406](https://github.com/gridstatus/gridstatus/pull/406)
+* Add scrapers for PJM wind and solar generation zonal data [#404](https://github.com/gridstatus/gridstatus/pull/404)
+* Add ERCOT AS Plan [#400](https://github.com/gridstatus/gridstatus/pull/400)
+* Adds scrapers for three additional PJM datasets [#401](https://github.com/gridstatus/gridstatus/pull/401)
+* Add PJM 5 minute solar generation and instantaneous wind generation [#399](https://github.com/gridstatus/gridstatus/pull/399)
+* Add ERCOT API for Hourly Solar and Wind Reports [#388](https://github.com/gridstatus/gridstatus/pull/388)
+* ERCOT LMP by Bus Using New API [#386](https://github.com/gridstatus/gridstatus/pull/386)
+* Add Real Time Adders and Reserves for ERCOT [#383](https://github.com/gridstatus/gridstatus/pull/383)
+* Add ERCOT Temperature Forecasts by Weather Zone [#384](https://github.com/gridstatus/gridstatus/pull/384)
+* Add Short Term System Adequacy [#382](https://github.com/gridstatus/gridstatus/pull/382)
+* add gen_outages_by_type dataset [#381](https://github.com/gridstatus/gridstatus/pull/381)
+
+### Fixes
+* IESO HOEP Real Time Fix Duplicates [#576](https://github.com/gridstatus/gridstatus/pull/576)
+* MISO Zonal Load Numeric Cols [#568](https://github.com/gridstatus/gridstatus/pull/568)
+* ERCOT SCED Timestamp Column Rename Fix [#562](https://github.com/gridstatus/gridstatus/pull/562)
+* Keep reserve type [#554](https://github.com/gridstatus/gridstatus/pull/554)
+* EIA Generators Fix Data Types [#549](https://github.com/gridstatus/gridstatus/pull/549)
+* Fix ERCOT 60 Day DAM Columns [#538](https://github.com/gridstatus/gridstatus/pull/538)
+* ERCOT Fix Real Time AS Monitor, Real Time System Conditions, and Forecasts Publish Dates DST Issue [#468](https://github.com/gridstatus/gridstatus/pull/468)
+* DST Fix for energy_storage_resources [#469](https://github.com/gridstatus/gridstatus/pull/469)
+* ERCOT DAM LMP By Bus DST Fix [#467](https://github.com/gridstatus/gridstatus/pull/467)
+* Add small fix to ERCOT temp method [#463](https://github.com/gridstatus/gridstatus/pull/463)
+* MISO Outages Fix for Missing Columns [#459](https://github.com/gridstatus/gridstatus/pull/459)
+* Fix ISONE API methods [#454](https://github.com/gridstatus/gridstatus/pull/454)
```

Comment:
0.29.1 was released on January 27, 2025. Most of these PRs are from prior to that date.